test/functional/test_runner.py

```diff
@@ -708,14 +709,16 @@ def _get_uncovered_rpc_commands(self):
         Return a set of currently untested RPC commands.
 
         """
-        # This is shared from `test/functional/test-framework/coverage.py`
+        # This is shared from `test/functional/test_framework/coverage.py`
         reference_filename = 'rpc_interface.txt'
         coverage_file_prefix = 'coverage.'
 
         coverage_ref_filename = os.path.join(self.dir, reference_filename)
         coverage_filenames = set()
         all_cmds = set()
-        covered_cmds = set()
+        # Consider RPC generate covered, because it is overloaded in
+        # test_framework/test_node.py and not seen by the coverage check.
+        covered_cmds = set({'generate'})
```

Comment:
I don't quite understand this. `generate` is not returned by `help` and so I don't think its even added to the coverage list:

https://github.com/bitcoin/bitcoin/blob/bd00d3b1f2036893419d1e8c514a8af2c4e4b1fb/test/functional/test_framework/coverage.py#L96

Both with and without this change, my coverage results were the same:

```
Uncovered RPC commands:
  - pruneblockchain

```

Reply:
@pinheadmz IIRC this was needed to appease a failing check in the Travis CI.

Reply:
@pinheadmz Yes, it's a hidden command (not in the help, see the added test) as suggested by @laanwj. IIRC this was needed to appease a failing check in the Travis CI that despite the added test it thought there was no coverage.